scalatest/src/main/scala/org/scalatest/prop/Randomizer.scala

```diff
@@ -166,11 +175,14 @@ class Randomizer(private[scalatest] val seed: Long) { thisRandomizer =>
     * @return A random Double in that range, and the next Randomizer to use.
     */
   def nextDoubleBetween0And1: (Double, Randomizer) = {
+    // TODO: Verify this implementation against the spec and comment it.
     val (ia, ra) = thisRandomizer.next(26)
     val (ib, rb) = ra.next(27)
     (((ia.toLong << 27) + ib) / (1L << 53).toDouble, rb)
   }
 
+   // TODO: Any reason why we have both a next and a nextInt? Should we drop next?
```

Comment:
I don't understand -- `next()` is the method that underpins all the `nextXX()` functions.  How would you drop it?

Reply:
Yes I realized next takes bits and nextInt does not, so I deleted this comment in my later PR.